code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -79,6 +79,13 @@
 	end_sound = 'sound/machines/microwave/microwave-end.ogg'
 	volume = 90
 
+/datum/looping_sound/lathe_print
+	start_sound = 'sound/machines/lathe/protolathe_print.ogg'
+	start_length = 15
+	mid_sounds = list('sound/machines/lathe/protolathe_print2.ogg' = 1)
+	mid_length = 20
+	vary = TRUE
```

Comment:
This is assuming you keep with these sounds, since they are cool but over a long period they're maybe not so positive. Goal is to provide a vibe, but s important that sounds are not so opinionated that they control a whole room. Background music you know?